design/schema/bcdc.puml

```diff
@@ -0,0 +1,319 @@
+    url: String!
+    rrid: String
+}
+
+class Project {
+    * id: String!
+    doi: String
+    handle: String!
+    title: String!
+    shortTitle: String!
+    description: String!
+    bkpWebResource: WebResource!
+    informationWebResource: WebResource
+    subprogram: [Subprogram]!
+    technique: [Technique]!
+    collection: [Collection]!
+    specimenType: [SpecimenType]!
+    contact: Contact
+    creator: [Agent]!
+    fundingSource: [Award]!
+    contributor: [Agent]
+    highlightedWebResource: [WebResource]
+    protocol: [Protocol]
+    dataArchive: [Organization]
+    externalId: String
+    species: [Species]!
+    modality: [Modality]!
+    publication: [Publication]
+    dateModified: Date
+    dateCreated: Date!
+}
+
+class Collection {
+    * id: String!
+    handle: String!
+    title: String!
+    shortTitle: String!
+    description: String!
+    doi: String
+    rrid: String
+    citation: String
+    externalId: String
+    dataArchive: Organization
+    bkpWebResource: [WebResource]!
+    dataArchive: [Organization]!
+    species: [Species]!
+    specimenCount: [SpecimenCount]!
+    license: License!
+    accessControl: AccessControl!
+    completionState: CompletionState!
+    modality: [Modality]!
+    technique: [Technique]!
+    webResource: [WebResource]
+    dateModified: Date
+    dateCreated: Date!
+}
+
+class Subprogram {
+    * id: String!
+    doi: String
+    handle: String!
+    title: String!
+    shortTitle: String!
+    description: String!
+    bkpWebResource: WebResource
+    program: Program!
+    informationWebResource: WebResource
+}
+
+class Program {
+    * id: String!
+    doi: String
+    handle: String!
+    title: String!
+    shortTitle: String!
+    description: String!
+    bkpWebResource: WebResource
+    informationWebResource: WebResource
+}
+
+class Protocol {
+    * id: String!
+    handle: String
+    title: String!
+    shortTitle: String!
+    webResource: WebResource!
+}
+
+class Person {
+    * id: String!
+    name: String!
+    givenName: String!
+    familyName: String!
+    handle: String
+    orcid: String
+}
+
+class Organization {
+    * id: String
+    name: String!
+    handle: String!
+    ror: String
+    webResource: WebResource
+    contact: Contact
+}
+
+class License {
+    * id: String!
+    handle: String
+    name: String!
+    title: String!
+    shortTitle: String!
+    webResource: WebResource!
+}
+
+class Award {
+    * id: String
+    name: String!
+    title: String!
+    identifier: String!
+    reportSymbol: String
+    funder: Agent!
+    awardee: [Agent]!
+}
+
+class Technique {
+    * id: String!
+    handle: String
+    name: String!
+}
+
+class Species {
+    * id: String!
+    name: String!
+    common_name: String!
+    scientific_name: String
+    ncbi_taxonomy_id: String!
+    rrid: String
+}
+
+class SpecimenCount {
+    * id: String!
+    count: Int
+    specimenType: SpecimenType!
+}
+
+class Contact {
+    * id: String!
+    name: String!
```

Comment:
how does this interact with person?

Reply:
This was polymorphic, with contact being either an organization or person. Updated to be a direct link to Person, with a circle for contact for readability.